### PR TITLE
Sort out GHA set-output deprecation warnings

### PR DESCRIPTION
**Pull request by clydetealium**
This pr addresses the following:
- the GHA set-output deprecation warning
- the lack of a versioned distribution build

See the following for more information on GHA set-output deprecation:
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.org
```
Comment (raynigon) [inline .npmrc:1]: This file should be removed from this PR

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  target: 'node',
+  mode: 'development',
+  optimization: {
+    minimize: false
+  },
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'index.js',
+  },
+};
```
Comment (raynigon) [inline webpack.config.js:14]: Newline should be added

**Review by raynigon — CHANGES_REQUESTED** [review]

**Comment by yonatanaftali** [comment]
@clydetealium Thanks for your contribution! Any update on this?

**Comment by thsaravana** [comment]
Sorry for the delay. I merged the #41 PR that had the same changes. Thanks @clydetealium !

File: webpack.config.js (hunk 2 above)
Comment (thsaravana) [reply webpack.config.js:14]: Merged. I will create a v1.4 tag once I review the other PR's as well.

**Pull request closed without merging**